saleor/graphql/plugins/mutations.py

```diff
@@ -99,7 +99,9 @@ def perform_mutation(cls, _root, info: ResolveInfo, /, **data):
         plugin_id = cleaned_data["plugin"].PLUGIN_ID
         channel_slug = cleaned_data["channel_slug"]
         input_data = cleaned_data["data"]
+
         manager = get_plugin_manager_promise(info.context).get()
         manager.save_plugin_configuration(plugin_id, channel_slug, input_data)
-        manager = get_plugins_manager()
+
+        manager = get_plugins_manager(allow_replica=False)
```

Comment:
@8r2y5 One more thing, I checked with @fowczarek, and all DB calls in mutation should be done using writer. Can you verify that both calls to `get_plugin_manager_promise` are using the writer here? I think this will require enabling replicas locally.

Reply:
Tested locally with replica enabled, it is selecting writer when using `get_plugin_manager_promise`